### PR TITLE
Added an installation for WSL

### PR DESCRIPTION
**Pull request by icm7216**
Added an installation for WSL Ubuntu-20.04(Windows) + rbenv

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -54,6 +54,29 @@ license/lgpl-3.0.txtを見てください。
   % rackup > /dev/null 2>&1 &
   % open http://localhost:9292/
 
+==== WSL Ubuntu-20.04(Windows) + rbenv
+
+First, install groonga in advance.
```
Comment (kou) [inline README.rdoc:59]:
```suggestion
First, install Groonga in advance.
```

File: README.rdoc [hunk 2]
```diff
@@ -54,6 +54,29 @@ license/lgpl-3.0.txtを見てください。
...
+groonga installation is easy and convenient to {install from Ubuntu PPA.}[https://groonga.org/ja/docs/install/ubuntu.html#ubuntu]
```
Comment (kou) [inline README.rdoc:60]:
```suggestion
Groonga installation is easy and convenient to {install from Ubuntu PPA}[https://groonga.org/ja/docs/install/ubuntu.html#ubuntu].
```

File: README.rdoc [hunk 3]
```diff
@@ -54,6 +54,29 @@ license/lgpl-3.0.txtを見てください。
...
+
+Enter in the terminal of WSL
```
Comment (kou) [inline README.rdoc:62]:
```suggestion
Enter the following command lines in the terminal of WSL:
```

File: README.rdoc [hunk 4]
```diff
@@ -54,6 +54,29 @@ license/lgpl-3.0.txtを見てください。
...
+  $ git clone https://github.com/ranguba/racknga.git
+  $ git clone https://github.com/clear-code/rurema-search.git
+  $ rbenv local 2.6.6
+  $ cd rurema-search
+  $ gem install groonga
+  $ bundle install
+  $ ./update.sh
+  $ rackup
+
+Finally, open a browser http://localhost:9292/
```
Comment (kou) [inline README.rdoc:77]:
```suggestion
Finally, open http://localhost:9292/ by your Web browser.
```

**Comment by kou** [comment]
Wow!
I didn't know this works on WSL.

**Comment by icm7216** [comment]
Yes, rurema-search works very well on WSL. 
Thank you for your review.

**Pull request merged**